ipaserver/plugins/agreements.py

```diff
@@ -0,0 +1,234 @@
+import six
+
+from ipalib import api, errors, output
+from ipalib import Bool, Str, StrEnum
+from ipalib.plugable import Registry
+from .baseldap import (
+    LDAPObject, LDAPSearch, LDAPCreate, LDAPDelete,
+    LDAPUpdate, LDAPRetrieve, LDAPAddMember, LDAPRemoveMember,
+    global_output_params, pkey_to_value)
+from .hbacrule import is_all
+from ipalib import _, ngettext
+from ipapython.dn import DN
+
+if six.PY3:
+    unicode = str
```

Comment:
please don't copy this bad style, use str()

Reply:
fixed with https://github.com/fedora-infra/freeipa-fas/pull/105/commits/abe8b38526e5a8bd74dbcb65acf7a01a5d45cb44